src/layouts/Catalogue/Catalogue.js

```diff
@@ -188,7 +188,7 @@ export default class Catalogue extends Component {
             <h2>{this.state.selectedProduct.title}</h2>
             <div style={{ fontSize: "12px" }}>
               {this.state.selectedProduct.description}<br/>
-              <p><strong>${this.state.selectedProduct.price}</strong></p>
+              <strong>${this.state.selectedProduct.price}</strong>
             </div>
```

Comment:
The `div` should be a `p`. And if you give the `strong` a `display: block` you won't need the `<br/>` anymore.

Reply:
Although the styling issue is better to solve in the Styled Components assignment.